transcom/mymove
# B 19889 UI estimatedprice int

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19889)

This branch does have commits from [B-19888](https://github.com/transcom/mymove/pull/12814/commits) in it.

To test:
1. Run `make tasks_save_ghc_fuel_price_data`. If you don't run this, you won't see a price for the Fuel Surcharge (FSC) service item.
2. Create a shipment in the prime sim, and make sure to provide an estimated weight.
3. Approve the shipment as the TOO.
4. Check the MTO page and scroll down to the service item section. You should see the estimated price for the service items, as long you provided an estimated weight as the prime.
5. For DLH, DSH, FSC , DOP, DDP, DPK, and DUPK, you should see prices.

6. Make another shipment, without providing an estimated weight.
7. Approve as TOO, check out the MTO page. You should see "Estimated Price: -" instead of prices for the above service items.

## Screenshots

![image](https://github.com/transcom/mymove/assets/147537467/c096bfdc-54ba-4c1f-8f53-3c95c52f692e)

![image](https://github.com/transcom/mymove/assets/147537467/906d9409-3abd-4e5c-9ece-26d183e0c032)
